R/methods-epi_df.R

```diff
@@ -67,15 +67,15 @@ summary.epi_df = function(object, ...) {
 #' @export
 #' @noRd
 head.epi_df = function(x, ...) {
-  head(tibble::as_tibble(x), ...)
+  as_epi_df(NextMethod("head"))
```

Comment:
Minor "style"-like thing: might want to exclude `"head"` if it's not necessary to match how `NextMethod` is called elsewhere in the file, or explain why it's necessary in a comment if it is necessary.

Reply:
Added comment as to why "head" was specified in NextMethod() in file (essentially, the first arg. should be the name of the generic fun. that want to offload the work to).